controllers/kogitoserverlessplatform_controller.go

```diff
@@ -69,7 +69,7 @@ func (r *KogitoServerlessPlatformReconciler) Reconcile(ctx context.Context, req
 	if ok, err := platform.IsOperatorAllowedOnNamespace(ctx, r.Reader, req.Namespace); err != nil {
 		return reconcile.Result{}, err
 	} else if !ok {
-		logger.Info("Ignoring request because namespace is locked")
+		logger.Info(fmt.Sprintf("Ignoring request because the operator hasn't got the permissions to work on namespace %s", req.Namespace))
```

Comment:
maybe a warning level would fit better here.

Reply:
The logr library hasn't got the "warning" level (there are only info and error), but we can change the verbosity level.

Also into the logger utility that we have got into the [container-builder](https://github.com/kiegroup/container-builder/blob/96342f7cc7cbea59f1cf1f0ec941c205a2b27e6a/util/log/log.go) there isn't a warning level.

We could switch to the logger utility that there is into the operator-utils as soon as we will have an updated version we can import into the operator.